editor/editor_node.cpp

```diff
@@ -151,6 +150,49 @@ EditorNode *EditorNode::singleton = nullptr;
 // The metadata key used to store and retrieve the version text to copy to the clipboard.
 static const String META_TEXT_TO_COPY = "text_to_copy";
 
+class AcceptDialogAutoReparent : public AcceptDialog {
```

Comment:
Will this work correctly with all virtual and ML calls without the GDCLASS macro?

Reply:
Nothing here requires it, but it should be there in case something will need it in the future. Fixed.